website/docs/language/expressions/types.mdx

```diff
@@ -175,7 +175,8 @@ expression must produce a value of that type.
 Where possible, Terraform automatically converts values from one type to
 another in order to produce the expected type. If this isn't possible, Terraform
 will produce a type mismatch error and you must update the configuration with a
```

Comment:
```suggestion
another to produce the expected type. If this conversion is impossible, Terraform
produces a type mismatch error and you must update the configuration with a
```

I believe this was the suggestion that was missed in https://github.com/hashicorp/terraform/pull/31706/files#r964849485